tests/onnxruntime/test_modeling.py

```diff
@@ -3097,11 +3098,13 @@ def test_load_vanilla_transformers_which_is_not_supported(self):
 
     @parameterized.expand(grid_parameters({"model_arch": SUPPORTED_ARCHITECTURES, "use_cache": [True]}))
     def test_generate_utils(self, test_name: str, model_arch: str, use_cache: str):
+        if model_arch == "encoder-decoder" and use_cache is True:
+            self.skipTest("encoder-decoder model type with use_cache=True is not supported for bert as a decoder")
```

Comment:
same question with cache

Reply:
same as above, check this comment [discussion_r1162759972](https://github.com/huggingface/optimum/pull/851#discussion_r1162759972)